packages/next/src/server/instrumentation/types.ts

```diff
@@ -6,6 +6,7 @@ export type RequestErrorContext = {
     | 'react-server-components'
     | 'react-server-components-payload'
     | 'server-rendering'
+  revalidateReason?: 'on-demand' | 'stale' | undefined
```

Comment:
Are we able to make this non-optional?
```suggestion
  revalidateReason: 'on-demand' | 'stale' | undefined
```